sources/packages/backend/apps/api/src/route-controllers/announcements/announcement.institutions.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller, Get, Query } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { ApiTags } from "@nestjs/swagger";
+import { Announcement } from "@sims/sims-db";
+import { AnnouncementService } from "../../services";
+import { InstitutionAnnouncementsAPIInDTO } from "../../route-controllers/announcements/models/announcement.dto";
+import { AllowAuthorizedParty } from "../../auth/decorators";
+import { AuthorizedParties } from "../../auth";
+import { ClientTypeBaseRoute } from "../../types";
+
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("announcements")
+@ApiTags(`${ClientTypeBaseRoute.Institution}-announcements`)
+export class AnnouncementInstitutionsController extends BaseController {
+  constructor(private readonly announcementService: AnnouncementService) {
+    super();
+  }
+
+  /**
+   * Get system announcements.
+   * @param target location for the announcement.
```

Comment:
Please fix the comment here param name and description must be updated to reflect code change.

Reply:
updated